scraper.rb

```diff
@@ -59,8 +59,14 @@ class MemberPage < Page
     url.to_s.split('/').last
   end
 
+  TITLE_RE = /^(Hon\.|Sen\.) /
+
   field :name do
-    box.xpath('.//th[text()="Name"]/following-sibling::td').text.tidy.sub('Hon. ', '')
+    name_with_title.gsub(TITLE_RE, '')
+  end
+
+  field :title do
+    $1 if name_with_title.match(TITLE_RE)
```

Comment:
This line fails `rubocop`, which it seems has never actually been set to auto-run in this repo (I've now fixed that). 

The autocorrect mode in `rubocop` would turn that into 

```ruby
Regexp.last_match(1) if name_with_title.match(TITLE_RE)
```

But I think that's fairly ugly. More in keeping with house style would be:

```ruby
name_with_title[TITLE_RE, 1]
```